go/install/install_windows.go

```diff
@@ -374,7 +374,16 @@ func LsofMount(mountDir string, log Log) ([]CommonLsofResult, error) {
 	return nil, fmt.Errorf("Cannot use lsof on Windows.")
 }
 
+// delete this function and calls to it if present after 2022
+func deleteDeprecatedFileIfPresent() {
+	// this file is no longer how we do things, and if it's present (which it shouldn't be) it could
+	// cause unexpected behavior
+	autostartLinkPath := filepath.Join(appDataDir, "Microsoft\\Windows\\Start Menu\\Programs\\Startup\\KeybaseStartup.lnk")
+	_ = os.Remove(autostartLinkPath)
+}
+
 func GetAutostart(context Context) keybase1.OnLoginStartupStatus {
+	deleteDeprecatedFileIfPresent()
```

Comment:
This might be called more often than once per run, but that might be fine.

Reply:
yeah - agreed. not a huge deal.